src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import static java.lang.Math.PI;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        setColor(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return PI * radius * radius;
```

Comment:
It's better not to use static imports if possible
```suggestion
        return Math.PI * radius * radius;
```